easybuild/easyconfigs/a/at-spi2-core/at-spi2-core-2.30.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'MesonNinja'
+
+name = 'at-spi2-core'
+version = '2.30.0'
+
+homepage = 'https://www.freedesktop.org/wiki/Accessibility/AT-SPI2/'
+description = """
+At-Spi2-core includes the protocol definitions for the new D-Bus at-spi.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+sources = [SOURCELOWER_TAR_XZ]
+source_urls = [FTPGNOME_SOURCE]
+checksums = ['0175f5393d19da51f4c11462cba4ba6ef3fa042abf1611a70bdfed586b7bfb2b']
+
+builddependencies = [
+    ('Meson', '0.48.1', '-Python-3.6.6', ('foss', '2018b')),
+    ('Ninja', '1.8.2', '', ('foss', '2018b')),
```

Comment:
Same problem as in #7401, can we avoid introducing build dependencies that use the `foss` toolchain while the toolchain used here is `GCCcore`?